src/qt/fontenum.cpp

```diff
@@ -7,13 +7,23 @@
 
 // For compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
-
 #include "wx/fontenum.h"
+#include <QFontDatabase>
 
-bool wxFontEnumerator::EnumerateFacenames( wxFontEncoding WXUNUSED(encoding),
-                bool WXUNUSED(fixedWidthOnly))
+bool wxFontEnumerator::EnumerateFacenames( wxFontEncoding WXUNUSED(encoding), bool fixedWidthOnly)
 {
-    return false;
+    QFontDatabase fontDatabase;
+    const QStringList allFonts = fontDatabase.families(QFontDatabase::Any);
+    for( QStringList::const_iterator i = allFonts.begin(); i != allFonts.end(); ++i)
+    {
+        const QString &fontFamily = *i;
```

Comment:
```suggestion
        const QString& fontFamily = *i;
```

Reply:
I'll apply these changes when I fix up the include paths and build error